src/styles/index.scss

```diff
@@ -1,3 +1,113 @@
+@import '../utils/variables';
+@import './reset.css';
```

Comment:
According to the Code Style Rules, we should use SCSS files as imports. So, you should convert your reset.css file to SCSS format and import it as a SCSS file.